### PR TITLE
Update more links

### PR DESCRIPTION
**Pull request by 1chandu**
For #

#### Background
Update more github urls to new visgl repo.

uber.github.io/luma.gl   ->    visgl.github.io/luma.gl
uber.github.io/react-map-gl -> visgl.github.io/react-map-gl
github.com/uber/kepler.gl -> github.com/keplergl/kepler.gl

Also fixed following urls (not directly related moving to visgl repo.)

/luma.gl/#/documentation/ -> /luma.gl/docs/

#### Change List
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +13,11 @@
   </a>
 </p>
 
-<h1 align="center">deck.gl | <a href="https://uber.github.io/deck.gl">Website</a></h1>
+<h1 align="center">deck.gl | <a href="https://visgl.github.io/deck.gl">Website</a></h1>
```
Comment (Pessimistress) [inline README.md:16]: Use https://deck.gl

File: docs/whats-new.md [hunk 2]
```diff
@@ -397,7 +397,7 @@ The following table compares the performance between CPU and GPU aggregations us
...
+For React users, it is now easy to use [react-map-gl](http://visgl.github.io/react-map-gl/#/Examples/markers-popups) components with DeckGL, including DOM-based [marker](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/marker), [popup](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/popup), [navigation control](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/navigation-control), [fullscreen control](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/fullscreen-control) and [geolocate control](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/geolocate-control). This can be done by supplying the `ContextProvider` prop on `DeckGL`:
```
Comment (Pessimistress) [inline docs/whats-new.md:400]: These links are outdated

File: docs/whats-new.md [hunk 3]
```diff
@@ -1109,11 +1109,11 @@ In addition, the `queryObject` offers a much requested `radius` parameter, allow
 
 ### Shader Assembly
 
-For developers that write their own custom layers, the `shadertools` shader assembly system is now in place to replace the existing `assembleShaders` function in deck.gl. The new shader assembler system supports organizing shader code into modules and is integrated with luma.gl's [`Model`](http://uber.github.io/luma.gl/#/documentation/api-reference/model) so users no longer need to call `assembleShaders` before creating the `Model` for the layer.
+For developers that write their own custom layers, the `shadertools` shader assembly system is now in place to replace the existing `assembleShaders` function in deck.gl. The new shader assembler system supports organizing shader code into modules and is integrated with luma.gl's [`Model`](http://visgl.github.io/luma.gl/#/documentation/api-reference/model) so users no longer need to call `assembleShaders` before creating the `Model` for the layer.
```
Comment (Pessimistress) [inline docs/whats-new.md:1112]: Same

File: docs/whats-new.md [hunk 4]
```diff
@@ -1109,11 +1109,11 @@ In addition, the `queryObject` offers a much requested `radius` parameter, allow
...
 
 ### Per-Layer Control of WebGL Parameters
 
-The base `Layer` class (which is inherited by all layers) supports a new property `parameters` that allows applications to specify the state of WebGL parameters such as blending mode, depth testing etc. This provides applications with significant control over the detailed rendering of layers. Note that the new `parameters` prop directly leverages the luma.gl v4 [setParameters](http://uber.github.io/luma.gl/#/documentation/api-reference/get-parameter) API, which allows all WebGL parameters to be specified as keys in a single parameter object.
+The base `Layer` class (which is inherited by all layers) supports a new property `parameters` that allows applications to specify the state of WebGL parameters such as blending mode, depth testing etc. This provides applications with significant control over the detailed rendering of layers. Note that the new `parameters` prop directly leverages the luma.gl v4 [setParameters](http://visgl.github.io/luma.gl/#/documentation/api-reference/get-parameter) API, which allows all WebGL parameters to be specified as keys in a single parameter object.
```
Comment (Pessimistress) [inline docs/whats-new.md:1116]: Same

File: docs/whats-new.md (hunk 2 above)
Comment (1chandu) [reply docs/whats-new.md]: Looks these links refer to old react-map-gl website demos, looks like lot of things changed, not sure what the correct links are now, I don't have the context here.

Comment (tsherif) [reply docs/whats-new.md]: For the what's new in luma.gl, if the links were no longer valid (e.g. old lesson examples), I just removed them.

**Pull request merged**